### PR TITLE
APIS-6516

### PR DESCRIPTION
**Pull request by peteslater-ee**

### CODE REVIEW

File: test/uk/gov/hmrc/apidefinition/service/AwsAPIPublisherSpec.scala [hunk 1]
```diff
@@ -85,8 +85,8 @@ class AwsAPIPublisherSpec extends AsyncHmrcSpec {
       val swaggerDetailsCaptor: ArgumentCaptor[AWSSwaggerDetails] = ArgumentCaptor.forClass(classOf[AWSSwaggerDetails])
       when(underTest.awsAPIPublisherConnector.createOrUpdateAPI(*, swaggerDetailsCaptor.capture())(*))
         .thenReturn(successful(UUID.randomUUID().toString))
-      val apiDefinition1: ApiDefinition                           = someAPIDefinition("API 1")
-      val apiDefinition2: ApiDefinition                           = someAPIDefinition("API 2")
+      val apiDefinition1: StoredApiDefinition                     = someAPIDefinition("API 1")
```
Comment (johnsgp) [inline test/uk/gov/hmrc/apidefinition/service/AwsAPIPublisherSpec.scala:88]: Perhaps these variables could change to `storedApiDefinition1` and `...2`?

File: test/uk/gov/hmrc/apidefinition/service/AwsAPIPublisherSpec.scala [hunk 2]
```diff
@@ -102,7 +102,7 @@ class AwsAPIPublisherSpec extends AsyncHmrcSpec {
       val swaggerDetailsCaptor: ArgumentCaptor[AWSSwaggerDetails] = ArgumentCaptor.forClass(classOf[AWSSwaggerDetails])
       when(underTest.awsAPIPublisherConnector.createOrUpdateAPI(*, swaggerDetailsCaptor.capture())(*))
         .thenReturn(successful(UUID.randomUUID().toString))
-      val apiDefinition: ApiDefinition                            = someAPIDefinition()
+      val apiDefinition: StoredApiDefinition                      = someAPIDefinition()
```
Comment (johnsgp) [inline test/uk/gov/hmrc/apidefinition/service/AwsAPIPublisherSpec.scala:105]: Rename variable and method to have `stored` in their names?

File: test/uk/gov/hmrc/apidefinition/service/DocumentationServiceSpec.scala [hunk 3]
```diff
@@ -79,7 +79,7 @@ class DocumentationServiceSpec extends AsyncHmrcSpec with GuiceOneAppPerSuite wi
     authorised = false
   )
 
-  val apiDefinition: ApiDefinition = ApiDefinition(
+  val apiDefinition: StoredApiDefinition = StoredApiDefinition(
```
Comment (johnsgp) [inline test/uk/gov/hmrc/apidefinition/service/DocumentationServiceSpec.scala:82]: Variable name could be `storedApiDefinition` then don't need the type as well.

**Review by johnsgp — APPROVED** [review]

File: test/uk/gov/hmrc/apidefinition/service/AwsAPIPublisherSpec.scala (hunk 1 above)
Comment (peteslater-ee) [reply test/uk/gov/hmrc/apidefinition/service/AwsAPIPublisherSpec.scala:88]: We think they are OK as they are

File: test/uk/gov/hmrc/apidefinition/service/DocumentationServiceSpec.scala (hunk 3 above)
Comment (peteslater-ee) [reply test/uk/gov/hmrc/apidefinition/service/DocumentationServiceSpec.scala:82]: It's fine

**Review by AndySpaven — APPROVED** [review]

**Pull request merged**